tests/plugins/semanticdb/test

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+. "$(dirname "$0")"/../../common.sh
+
+check_for_semanticdb_files() {
+	for filename in "A.scala.semanticdb" "B.scala.semanticdb"; do
+		path="../../bazel-bin/plugins/semanticdb/_semanticdb/semanticdb-$1/META-INF/semanticdb/plugins/semanticdb/$filename"
+
+		if [ ! -f "$path" ]; then
+			echo "Error: $path doesn't exist"
+			exit 1
+		fi
+	done
+}
+
+bazel build :semanticdb-2_13
+check_for_semanticdb_files 2_13
+bazel build :semanticdb-3
+check_for_semanticdb_files '3'
```

Comment:
Nit: This has single quotes around it and the 2_13 version does not.

Reply:
Oh, I did that for syntax highlighting. It has no effect on the script. Bash differentiates numbers and strings, so it was being highlighted differently from `2_13`.